etc/spack/defaults/packages.yaml

```diff
@@ -62,6 +62,7 @@ packages:
       unwind: [libunwind]
       uuid: [util-linux-uuid, libuuid]
       wasi-sdk: [wasi-sdk-prebuilt]
+      xkbdata: [xkeyboard-config, xkbdata]
```

Comment:
We don't support having a virtual named as a real package. If this "works" is just by accident. Can you make both packages provide `xkbdata-api` and change all dependents?

Reply:
@teaguesterling I also noticed what @alalazo said is the case and was thinking of asking him to review this PR and suggest the proper change.

His suggestion to is of course the correct way, and changing the depends_on to -api is the way to go.